### ISSUE TITLE
Question: Importing the Library

### ISSUE THREAD

**Issue by 360disrupt** [Arachnid/solidity-stringutils#4]
I try to import it like in the Readme
```
import "github.com/Arachnid/solidity-stringutils/strings.sol";

contract Mortal {
   ...
```
But when I try to build it with `truffle` I get this:

```
Cannot find import strings.sol from Test.sol. If it's a relative path, ensure it starts with `./` or `../`.
Compilation failed. See above.
```
How to do import it?

**Comment by Arachnid** [comment]
How are you compiling this? github imports work fine in browser solidity, but if you're  compiling from the command line, you should download the library and put it somewhere the compiler can reach, updating the import path to match.

**Comment by 360disrupt** [comment]
Thx, I do compile it from the command line. So I probably need to check out if I configure a libraries directory in truffle?

**Comment by Arachnid** [comment]
Yes, whatever library support your framework has; putting it in the same directory will almost certainly work too.

**Comment by dustpan28645** [comment]
Great work
